common/http-api-client/Cargo.toml

```diff
@@ -39,3 +40,6 @@ features = ["tokio"]
 
 [dev-dependencies]
 tokio = { workspace = true, features = ["rt", "macros"] }
+
+[features]
+bincode = ["dep:bincode", "nym-http-api-common", "nym-http-api-common/bincode"]
```

Comment:
why not consistently use `dep:`?

Reply:
wanted to only add the prefix for the external dependency that was the source of the name clash, but I have no strong feelings about it